### ISSUE TITLE
Onboard Sonoma

### ISSUE THREAD

**Issue by lippytak** [codeforamerica/health#291]
Start with blurb/one pager to Shaydra.

**Comment by lippytak** [comment]
Bigwig buy in meeting scheduled for Aug 19th. Hopefully can get a CBO acct before.

**Comment by alanjosephwilliams** [comment]
what's next?

**Comment by alanjosephwilliams** [comment]
@daguar @lippytak should we loop back with Sonoma now that cranking up the volume is again a short/medium term goal?

**Comment by lippytak** [comment]
Good question/topic for offsite.
Personally I think we should be focused on making SD (and others) happy and not
reaching out to counties (especially non-Molina counties) right now.

On Mon, Jan 4, 2016 at 10:10 AM, Alan Joseph Williams < notifications@github.com > wrote:
@daguar @lippytak should we loop back with Sonoma now that cranking up the volume is again a
short/medium term goal?

**Comment by daguar** [comment]
I agree with you @lippytak — there wasn't a huge amount of excitement at the high level, so for now I think we focus where that exists (and/or where there's more strategic value, eg Molinda)
